app/models/order.rb

```diff
@@ -130,6 +130,7 @@ def clear_cart?
     order_details.destroy_all
```

Comment:
This method should probably not end with `?` - maybe `clear_cart` instead?

Reply:
I think it may be related to it being used as a guard. Not sure, but I see the rest also follow that pattern

Reply:
👍🏼 good to merge then